src/hotspot/share/opto/node.cpp

```diff
@@ -2329,6 +2329,13 @@ void Node_Array::sort( C_sort_func_t func) {
   qsort( _nodes, _max, sizeof( Node* ), func );
 }
 
+//-----------------------------------------------------------------------------
+void Node_Array::swap(uint i, uint j) {
```

Comment:
Use can use the swap method from globalDefinitions.hpp.

Reply:
Done, thanks! I also moved swapping to `PhaseIterGVN` to simplify the PR.